json-serde-cdh7-shim/pom.xml

```diff
@@ -15,43 +15,49 @@
 
     <name>json-cdh7-shim</name>
 
-<build>
-    <plugins>
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-jar-plugin</artifactId>
-      </plugin>
-	</plugins>
-  </build>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
+            </plugin>
+        </plugins>
+    </build>
 
     <dependencies>
-		<dependency>
-			<groupId>io.starburst.openjson</groupId>
-			<artifactId>openjson</artifactId>
-			<version>${openjson.version}</version>
-			<scope>compile</scope>
-		</dependency>
-
-	<dependency>
-	    <groupId>org.apache.hive</groupId>
-	    <artifactId>hive-serde</artifactId>
-	    <version>${cdh7.hive.version}</version>
-	    <scope>provided</scope>
-	</dependency>
-	<dependency>
-	    <groupId>org.apache.hive</groupId>
-	    <artifactId>hive-exec</artifactId>
-	    <version>${cdh7.hive.version}</version>
-	    <scope>provided</scope>
-	</dependency>
-
-	<dependency>
-	    <groupId>org.apache.hadoop</groupId>
-	    <artifactId>${hadoop.dependency}</artifactId>
-	    <version>${cdh7.hadoop.version}</version>
-	    <scope>provided</scope>
-	</dependency>
+        <dependency>
+            <groupId>io.starburst.openjson</groupId>
+            <artifactId>openjson</artifactId>
+            <version>${openjson.version}</version>
+            <scope>compile</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.hive</groupId>
+            <artifactId>hive-serde</artifactId>
+            <version>${cdh7.hive.version}</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.hive</groupId>
+            <artifactId>hive-exec</artifactId>
+            <version>${cdh7.hive.version}</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>${hadoop.dependency}</artifactId>
+            <version>${cdh7.hadoop.version}</version>
+            <scope>provided</scope>
+        </dependency>
     </dependencies>
+
+    <repositories>
+        <repository>
+            <id>Cloudera</id>
+            <name>Cloudera Maven Repo</name>
+            <url>https://repository.cloudera.com/artifactory/cloudera-repos/</url>
```

Comment:
cc @martint @electrum @kokosing

Reply:
We are asking for troubles here. We should use our repositories only and if needed lets copy cloudera artifacts into our repo.